os/board/common/crashdump.c

```diff
@@ -298,6 +419,11 @@ static int dump_via_uart(void)
 				break;
 			}
 		/* Check handshake string for TARGET REBOOT signal */
+		} else if (strncmp(host_buf, EXTFSDUMP_HANDSHAKE_STRING, strlen(EXTFSDUMP_HANDSHAKE_STRING)) == 0) {
```

Comment:
I think using `strlen(EXTFSDUMP_HANDSHAKE_STRING) + 1` is better for security issue.

Reply:
The very same was added base don earlier suggestions but ended up causing problems when comparing handshake strings for Mt7686_hdk.
So reverted it to strlen(<MACRO>) only